libc/build/lib/pgBackRestLibC/Build.pm

```diff
@@ -19,15 +19,16 @@ use lib dirname($0) . '/../lib';
 
 use pgBackRest::Common::Log;
 use pgBackRest::Common::String;
-use pgBackRest::Storage::Local;
```

Comment:
My comment is not about this line but of one that is not showing in this diff. In "sub buildPutDiffers" there is an openRead but there is no follow-up open() call so I am not understanding when a call to open is required and when it is not.

Reply:
This openRead() is from pgBackRestTest::Common::Storage which still follows the old Perl convention of opening in this function.  For testing, this is actually better since it requires fewer steps though it does not perform quite as well.